heron/config/src/yaml/conf/nomad/statemgr.yaml

```diff
@@ -0,0 +1,23 @@
+# local state manager class for managing state in a persistent fashion
+heron.class.state.manager: com.twitter.heron.statemgr.zookeeper.curator.CuratorStateManager
+
+# local state manager connection string
+heron.statemgr.connection.string:  127.0.0.1:2181
```

Comment:
why 2181? I think it is the default port for zk so maybe better to avoid if possible.

Reply:
2181 is the default port number for zookeeper.  If I was to run whole setup (scheduler, topology, and zookeeper) on a single node, this is the default I would use.  The cli and all executors will be able to contact the local zookeeper on 127.0.0.1:2181

Reply:
got it. thx.